From 45d545a7210418326328906beacfa6f292c9583c Mon Sep 17 00:00:00 2001
From: Duncan Macleod <duncan.macleod@ligo.org>
Date: Fri, 14 Jan 2022 11:34:36 +0000
Subject: [PATCH 02/12] cmake: set C_PYTHONARCHLIB for python

---
 build/cmake/CondorConfigure.cmake | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build/cmake/CondorConfigure.cmake b/build/cmake/CondorConfigure.cmake
index 7b40493d87..1dfab1a6ed 100644
--- a/build/cmake/CondorConfigure.cmake
+++ b/build/cmake/CondorConfigure.cmake
@@ -150,6 +150,11 @@ if(NOT WINDOWS)
 
 		set(PYTHON3_LIBRARIES "${PYTHON3_LIB}")
 
+		# where to install the Python library
+		set(C_PYTHON3ARCHLIB ${Python3_SITEARCH})
+		# ensure that this is a relative path
+		cmake_path(RELATIVE_PATH C_PYTHON3ARCHLIB BASE_DIRECTORY "${CMAKE_INSTALL_PREFIX}")
+
 		set(PYTHON3_INCLUDE_PATH "${PYTHON3_INCLUDE_DIRS}")
 		set(PYTHON3LIBS_VERSION_STRING "${PYTHON3_VERSION_STRING}")
 		set(PYTHON3_MODULE_SUFFIX "${PYTHON3_MODULE_EXTENSION}")
-- 
2.43.0

