From 369637c9a2d5157fc5031c5fdea03019fdb40f4c Mon Sep 17 00:00:00 2001
From: Duncan Macleod <duncan.macleod@ligo.org>
Date: Fri, 14 Jan 2022 11:37:08 +0000
Subject: [PATCH 04/12] man: set section 7 for classads man page

---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 039356ec65..5788e8f88f 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -157,7 +157,7 @@ pygments_style = 'colorful'
 # Short list of man pages to build which is used to add generice info
 MAN_PAGES = [
         ('classad_eval', 1),
-        ('classads', 1),
+        ('classads', 7),
         ('condor_adstash', 1),
         ('condor_advertise', 1),
         ('condor_check_userlogs', 1),
-- 
2.43.0

