From 9aee952a61348fd1f3ff26cfadec1ce7ace82a3c Mon Sep 17 00:00:00 2001
From: Duncan Macleod <duncan.macleod@ligo.org>
Date: Fri, 14 Jan 2022 11:36:01 +0000
Subject: [PATCH 05/12] python: don't link against libpython

---
 build/cmake/CondorConfigure.cmake | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/build/cmake/CondorConfigure.cmake b/build/cmake/CondorConfigure.cmake
index 1dfab1a6ed..f78c1d7ca3 100644
--- a/build/cmake/CondorConfigure.cmake
+++ b/build/cmake/CondorConfigure.cmake
@@ -148,7 +148,13 @@ if(NOT WINDOWS)
 
 		set(PYTHON3_EXECUTABLE        ${Python3_EXECUTABLE})
 
-		set(PYTHON3_LIBRARIES "${PYTHON3_LIB}")
+		# in conda builds we don't want to actually link against libpython
+		if (APPLE)
+			set(PYTHON3_LIBRARIES "-Wl,-undefined,dynamic_lookup")
+		else()
+			set(PYTHON3_LIBRARIES "")
+		endif()
+		set(PYTHON3LIBS_FOUND TRUE)
 
 		# where to install the Python library
 		set(C_PYTHON3ARCHLIB ${Python3_SITEARCH})
-- 
2.43.0

